src/scss/custom.scss

```diff
@@ -19,9 +19,10 @@
   --ifm-hover-overlay: rgba(93, 52, 242, 0.12);
   --ifm-code-font-size: 95%;
   --ifm-navbar-height: 64px;
-  --doc-sidebar-width: 400px !important;
+  --doc-sidebar-width:340px !important;
   --doc-sidebar-section-title: #c9c5d0;
   --logto-border: #c4c7c7;
+  --ifm-menu-link-sublist-icon: url('/static/img/side-bar-error.svg');
```

Comment:
file name looks weird. maybe `sidebar-collapse.svg`?

Reply:
good point no idea where does this name come from....

Reply:
lul